### ISSUE TITLE
Blog Landing Page QA

### ISSUE THREAD

**Issue by vickibrown** [thinkshout/thinkshout#820]
This QA is for the Blog landing page found here https://thinkshout.com/blog/

**Featured post at the top**
_The styles here seem to be linked to the listing below, we're going to have to separate them to make them both accessible._
- [x] Darken the image overlay by 10% (to start, we'll need to test this)
- [x] Change the title and author links to white

**Filter button**
_apply buttons styles to filter button_
- [x] Background color should be hex #F06D2F
- [x] text color of button in the to hex #242424
- [x] Add hover/focus state background color hex #F38958
- [x] Gotham medium 12 pixels

**Filter Dropdown**
- [x] Change background color to hex #626262
- [x] remove all caps from dropdown text options
- [x] Text should be gotham medium, 12 pixels

**Post Listings**
- [x] Change tag links to hex FFFFFF
- [x] Change tag link hover/focus state to just underline
- [x] Change Blog titles color to hex #F06D2F
- [x] Change author hover/focus to just underline

**Comment by JulesKhong** [comment]
@vickibrown  "Chosen" is being used to let us style the select, and it has a lot of accessibility bugs. When I tab to it, the label is read as "Blank". I can open it with my keyboard, but none of the dropdown options have accessible labels. We should look at using a native select for a quick fix or a different select option with accessible labels.

**Comment by komejo** [comment]
@JulesKhong I replaced chosen with the version of Nice Select I used on ChangeLab Solutions - I did a fair bit of work to make it more accessible, hoping we can re-use it here. https://a11y-2020--thinkshout.netlify.app/blog/

**Comment by komejo** [comment]
@vickibrown Ready for review at https://a11y-2020--thinkshout.netlify.app/blog/

The only thing that confused me about this was 'Change tag links to hex FFFFFF', which would be white, making them invisible? Set the to #000000 black for now. Let me know if we're talking about the same thing: 

![Screen Shot 2020-05-16 at 1 15 46 PM](https://user-images.githubusercontent.com/1668324/82129421-78b12880-9777-11ea-9b5c-3d608cae5954.png)

**Comment by vickibrown** [comment]
@komejo good catch, that is what I meant! Looks good, can we change all to "View all posts"?

**Comment by JulesKhong** [comment]
- [ ] Running WAVE, the select is missing a label. Could we add an aria-label with "Filter by topic" @vickibrown Does that work for you?
<img width="369" alt="Screen Shot 2020-05-28 at 3 12 17 PM" src="https://user-images.githubusercontent.com/19519395/83199304-c7ef4580-a0f5-11ea-9ea8-7e02046beae7.png">

**Comment by vickibrown** [comment]
@JulesKhong @komejo agree there should be a label, big oversight. Let's use the styles from the dates below to put above the filter.
<img width="670" alt="Screen Shot 2020-05-29 at 4 31 53 PM" src="https://user-images.githubusercontent.com/21184833/83313207-153ce700-a1ca-11ea-8630-40e254513f38.png">

**Comment by komejo** [comment]
@JulesKhong I worked on this a bit more and think I have a good fix in place - it's a little tiricksy, but seems to work really nicely with both tabbed browsing and voiceover. All other feedback applied as well, ready for final review on https://a11y-2020--thinkshout.netlify.app/blog/

**Comment by JulesKhong** [comment]
@komejo Well, your tricksy fix seems to be working with keyboard navigation and voiceover so I'm not mad at it :) This ticket is ready to go from an a11y QA perspective 👍

**Comment by vickibrown** [comment]
@komejo looks good to me too!
